CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ### 0.3.0 (Next)
 
 * Your contribution here.
-* [#20](https://github.com/dblock/danger-changelog/pull/21): Added support for custom '* Your contribution here.' line - [@antondomashnev](https://github.com/antondomashnev).
+* [#25](https://github.com/dblock/danger-changelog/pull/25): Add information how to configure the plugin in README.md - [@antondomashnev](https://github.com/antondomashnev).
```

Comment:
It makes sense, but we don't have any check in Danger to skip readme changes, do we?